pkgs/by-name/tl/tlock/package.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
+  pname = "tlock";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "eklairs";
+    repo = pname;
+    rev = "refs/tags/v${version}";
```

```suggestion
    rev = "v${version}";
```
Just `v${version}` should be fine.